### PR TITLE
docs(common): apply seo recommendations

### PR DESCRIPTION
**Pull request by Tsvetomir-Hr**
Related to: https://github.com/telerik/blazor/issues/10414

### CODE REVIEW

File: components/dropdownlist/overview.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ position: 0
 
 The Blazor DropDownList component allows the user to choose an option from a predefined set of choices presented in a dropdown list popup. The developer can control the [data]({%slug components/dropdownlist/databind%}), sizes, and various appearance options like class and [templates]({%slug components/dropdownlist/templates%}).
 
-## Creating the DropDownList
+## Creating the Blazor DropDownList
```
Comment (dimodi) [inline components/dropdownlist/overview.md:15]: (Check for broken links to this heading)

```suggestion
## Creating Blazor DropDownList
```

File: components/dropdownlist/overview.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#fil
 
 The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#grouping) [Read more about the Blazor DropDownList grouping...]({% slug components/dropdownlist/grouping %}).
 
-## Templates
+## Blazor DropDownList Templates
```
Comment (dimodi) [inline components/dropdownlist/overview.md:63]: This breaks the scanability and readability of the right navigation.

```suggestion
## Templates
```

File: components/dropdownlist/overview.md [hunk 3]
```diff
@@ -76,9 +76,9 @@ The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#gro
 
 @[template](/_contentTemplates/dropdowns/adaptive-rendering.md#intro)
 
-## Parameters
+## Blazor DropDownList Parameters
```
Comment (dimodi) [inline components/dropdownlist/overview.md:79]: The convention is:

```suggestion
## DropDownList Parameters
```

File: components/dropdownlist/overview.md [hunk 4]
```diff
@@ -140,7 +140,7 @@ The DropDownList provides the following popup settings:
 
 @[template](/_contentTemplates/dropdowns/features.md#popup-settings)
 
-## DropDownList Reference and Methods
+## Blazor DropDownList Reference and Methods
```
Comment (dimodi) [inline components/dropdownlist/overview.md:143]:
```suggestion
## DropDownList Reference and Methods
```

File: components/dropdownlist/overview.md [hunk 5]
```diff
@@ -183,7 +183,7 @@ By default, if no `Value` is provided and no `DefaultText` is defined, the DropD
 * To display a selected item when the component renders - provide the `Value` of the desired element. Note that it must match an item of the component's data source.
 
 
-## Examples
+## Blazor DropDownList Examples
```
Comment (dimodi) [inline components/dropdownlist/overview.md:186]:
```suggestion
## Examples
```

File: components/dropdownlist/filter.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ The default filter operator is `starts with`. You can choose a different operato
 
 ## Performance
 
-By default, the filtering is debounced with 150ms. Configure that with the [`FilterDebounceDelay`]({%slug components/dropdownlist/overview%}#parameters) parameter of the component.
+By default, the filtering is debounced with 150ms. Configure that with the [`FilterDebounceDelay`]({%slug components/dropdownlist/overview%}#blazor-dropdownlist-parameters) parameter of the component.
```
Comment (Tsvetomir-Hr) [inline components/dropdownlist/filter.md:29]:
```suggestion
By default, the filtering is debounced with 150ms. Configure that with the [`FilterDebounceDelay`]({%slug components/dropdownlist/overview%}#dropdownlist-parameters) parameter of the component.
```

**Review by dimodi — APPROVED** [review]

File: components/dropdownlist/overview.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Overview
 page_title: DropDownList Overview
-description: Explore the Blazor DropDownList component and see code examples. Read more in the Telerik UI documentation for Blazor.
+description: Blazor dropdownlist allows users to select an option from a list, enabling dynamic data binding and event handling in web apps.
```
Comment (yordan-mitev) [inline components/dropdownlist/overview.md:4]:
```suggestion
description: The Blazor DropDownList allows users to select an option from a list, enabling dynamic data binding and event handling in web apps.
```

File: components/dropdownlist/overview.md [hunk 8]
```diff
@@ -76,9 +76,9 @@ The Blazor DropDownList @[template](/_contentTemplates/dropdowns/features.md#gro
 
 @[template](/_contentTemplates/dropdowns/adaptive-rendering.md#intro)
 
-## Parameters
+## DropDownList Parameters
 
->caption The DropDownList provides various parameters that allow you to configure the component:
+>caption The Blazor DropDown list provides various parameters that allow you to configure the component:
```
Comment (yordan-mitev) [inline components/dropdownlist/overview.md:81]:
```suggestion
>caption The Blazor DropDownList provides various parameters that allow you to configure the component:
```

File: components/grid/export/excel.md [hunk 9]
```diff
@@ -206,4 +206,5 @@ The Excel export has the following specifics:
 
   * [Live Demo: Grid Excel Export](https://demos.telerik.com/blazor-ui/grid/export-excel)
   * [Custom cell formatting of the exported file with RadSpreadProcessing]({%slug grid-kb-custom-cell-formatting-with-radspreadprocessing%})
-  * [Custom cell formatting of the exported file with RadSpreadStreamProcessing]({%slug grid-kb-custom-cell-formatting-with-radspreadstreamprocessing%}) 
+  * [Custom cell formatting of the exported file with RadSpreadStreamProcessing]({%slug grid-kb-custom-cell-formatting-with-radspreadstreamprocessing%})
```
Comment (yordan-mitev) [inline components/grid/export/excel.md:209]: Use a consistent case style in the list&mdash;either title case or sentence case. This comment applies to all items in this list.

File: components/grid/filter/overview.md [hunk 10]
```diff
@@ -130,3 +130,4 @@ The Grid uses `Activator.CreateInstance<TItem>();` to obtain the item's type it
 * [Live Demo: Grid Filter Row](https://demos.telerik.com/blazor-ui/grid/filter-row)
 * [Live Demo: Grid Filter Menu](https://demos.telerik.com/blazor-ui/grid/filter-menu)
 * [Filter the Grid by date only]({%slug grid-kb-filter-date-only%})
```
Comment (yordan-mitev) [inline components/grid/filter/overview.md:132]:
```suggestion
* [Filter the Grid by Date Only]({%slug grid-kb-filter-date-only%})
```

File: components/grid/filter/searchbox.md [hunk 11]
```diff
@@ -150,3 +150,4 @@ The example below demonstrates all SearchBox settings in action, and also how to
 * [Search the Grid with a `StartsWith` operator]({%slug grid-kb-search-startswith%})
```
Comment (yordan-mitev) [inline components/grid/filter/searchbox.md:150]:
```suggestion
* [Search the Grid with a `StartsWith` Operator]({%slug grid-kb-search-startswith%})
```

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**